Fix and improve changing application icon for Windows page (#3959)
godotengine/godot-docs: getting_started/workflow/export/changing_application_icon_for_windows.rst

You will most likely want to change that for your project. There are two types
of icons that can be changed on Windows: the file icon and the taskbar icon.

Creating an ICO file
--------------------

> Windows does not use formats such as png or jpg for application icons, instead
> it uses a windows only format called ICO. You can create your application icon
> in any program, as long as you can bring that file into a program, such as GIMP,
> which can export an ICO file.

Reviewer: ```suggestion
Windows does not use formats such as png or jpg for application icons. Instead,
it uses a Windows-only format called ICO. You can create your application icon
in any program but you will have to convert it to an ICO file using a program such 
as GIMP.
```

> `This tutorial <https://www.youtube.com/watch?v=uqV3UfM-n5Y>`_ goes over how to
> export an ICO file with GIMP.

Reviewer: ```suggestion
`This video tutorial <https://www.youtube.com/watch?v=uqV3UfM-n5Y>`_ goes over how to
export an ICO file with GIMP.
```

.. image:: img/icon_taskbar_icon.png

To change the taskbar icon, go to
**Project → Project Settings → Application → Config → Windows Native Icon**.
Click on the folder icon and select your ICO file.

.. image:: img/icon_project_settings.png

> Doing this will only change the icon for windows. ``Macos Native Icon`` sets
> the icon for Mac. ``Icon`` sets the icon for everything else and the project
> manager.

Reviewer: ```suggestion
This setting only changes the icon for your exported game on Windows. 
To set the icon for macOS, use ``Macos Native Icon``. And for any other platform,
use the ``Icon`` setting.
```

.. image:: img/icon_result.png

> .. note:: if your icon isn't showing up properly try clearing the icon cache
>           by opening `Run` and using the command ``ie4uinit.exe -ClearIconCache``
>           or ``ie4uinit.exe -show`` on Windows 10

Reviewer: ```suggestion
.. note:: 
          
          if your icon isn't showing up properly, on Windows 10, try clearing the
          icon cache. To do so, open `Run` and call the command 
          ``ie4uinit.exe -ClearIconCache`` or ``ie4uinit.exe -show``.
```
